src/main/java/com/libertymutualgroup/herman/aws/ecs/broker/kms/KmsBroker.java

```diff
@@ -126,6 +129,14 @@ public String brokerKey(AWSKMS client, KmsAppDefinition definition, List<Tag> ta
                 String.format("Error updating key policy for key ID %s. Policy: %s", keyId, policy), ex);
         }
 
+        GetKeyRotationStatusResult statusResult = client.getKeyRotationStatus(new GetKeyRotationStatusRequest().withKeyId(keyId));
```

Comment:
I know it generally seems like no one would want to do this, but you might want to support disabling key rotation as well if someone really doesn't want it.